week4/week4-083.HangmanLogic/src/HangmanLogic.java

```diff
@@ -0,0 +1,56 @@
+
+public class HangmanLogic {
+
+    private String word;
+    private String guessedLetters;
+    private int numberOfFaults;
+
+    public HangmanLogic(String word) {
```

Comment:
Generalno, ovo mi se dopada jer pokazuje direktnu namjeru šta si htio i treba ovako kucati što se mene tiče, ali da znaš da se String i int unutar objekta po default-u inicijalizuju na 0 i na prazan string. :)